config/src/main/java/com/alibaba/nacos/config/server/model/GroupkeyListenserStatus.java

```diff
@@ -32,7 +32,7 @@
 
     private int collectStatus;
 
-    private Map<String, String> lisentersGroupkeyStatus;
+    private Map<String, String> listenersGrouperStatus;
```

Comment:
Why you change the name to 'listenersGrouperStatus', maybe 'listenersGroupKeyStatus' is better? I understand the word 'listener' is spelled wrong.

Reply:
yes,i will fix it;